src/Microsoft.Internal.IO/Microsoft.Internal.IO.sln

```diff
@@ -0,0 +1,29 @@
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.27213.1
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Internal.IO", "src\Microsoft.Internal.IO.csproj", "{FEF5811F-ED50-4407-A6B9-885EBD3206FB}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{E107E9C1-E893-4E87-987E-04EF0DCEAEFD}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Release|Any CPU = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{FEF5811F-ED50-4407-A6B9-885EBD3206FB}.Debug|Any CPU.ActiveCfg = net461-Windows_NT-Debug|Any CPU
```

Comment:
Needs an msbuild /t:UpdateVSConfigurations in the root.

Reply:
I'll run it again